src/dataBase/repository/books/bookRepo.go

```diff
@@ -40,27 +43,66 @@ func openDb() *sql.DB {
 	return db
 }
 
-func GetAll() ([]entity.Book, error){
+
+
+func GetByID(bookID int) (entity.Book, error) {
 	db:=openDb()
```

Comment:
in the dataBaseConfig we already have opened connectin
Besides, you dont close THIS connection 
To use heroku's connection, on the heroku itself, we must reconfigure dbConfig